install_files/ansible-base/roles/app-test/tasks/download_verify_tbb.yml

```diff
@@ -0,0 +1,74 @@
+---
+- name: Create tbb directory.
+  sudo: true
+  file:
+    path: "{{ tbb_directory }}"
+    state: directory
+    owner: "{{ non_default_securedrop_user }}"
+    group: "{{ non_default_securedrop_user }}"
+    mode: "0770"
+
+- name: Stat TBB tarball
+  stat:
+    path: "{{ tbb_directory }}/tor-browser-linux{{ tbb_arch }}-{{ tbb_release }}_{{ tbb_locale }}.tar.xz"
+  register: tbb_stat
+
+- name: Download TBB {{ tbb_release }} tarball and signature.
+  get_url:
+    url: "{{ item }}"
+    dest: "{{ tbb_directory }}/{{ item|basename }}"
+  with_items:
+    - https://dist.torproject.org/torbrowser/{{ tbb_release }}/tor-browser-linux{{ tbb_arch }}-{{ tbb_release }}_{{ tbb_locale }}.tar.xz
+    - https://dist.torproject.org/torbrowser/{{ tbb_release }}/tor-browser-linux{{ tbb_arch }}-{{ tbb_release }}_{{ tbb_locale }}.tar.xz.asc
+  when: not tbb_stat.stat.exists
+
+- name: Install gnupg-curl
```

Comment:
Hm.  I don't see an obvious place for it?  

`gnupg-curl` is only needed for the TBB install, so it seems like it could stay in that file (maybe at the top?).  Happy to move it if you have a suggestion though!